centaur/src/main/resources/standardTestCases/InitialWorkDirRequirement.inline_file.test

```diff
@@ -0,0 +1,15 @@
+name: inline_file
+testFormat: workflowsuccess
+workflowType: CWL
+workflowTypeVersion: v1.0
+
+files {
+  wdl: InitialWorkDirRequirement/inline_file.cwl
+}
+
+metadata {
+  "submittedFiles.workflowType": CWL
+  "submittedFiles.workflowTypeVersion": v1.0
+  # TODO CWL: Re-add this check when outputs are stable:
+  #"outputs.prime_list": "[2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61, 67, 71, 73, 79, 83, 89, 97]"
```

Comment:
If somebody knows how I can add this back please tell me! I hate that this is commented out

Reply:
You can do this to have predictable workflow names https://github.com/broadinstitute/cromwell/pull/3077/files#diff-aba1b9ae7445e5816fa7b4d4b1d2c171